## Service 
geometry_msgs/Pose2D waypoint
---
bool accepted